### DISCUSSION TITLE
Why is it that in the v4.7 is deprecated the vulnerability scan?

### DISCUSSION THREAD

**Discussion by styxiner** [wazuh/wazuh#22754] [Q&A]
So I'm searching in the docs to run a manual vulnerability scan and it says that in ver 4.7 it's deprecated, but in 4.6 it is not.
Why???

link to the docs:
https://documentation.wazuh.com/current/user-manual/api/reference.html#operation/api.controllers.vulnerability_controller.run_vulnerability_scan

**Comment by MiguelazoDS** [comment]
Hi @Jaime-Moranchel, 

The vulnerability detector module will change soon, the vulnerabilities will be indexed to the Wazuh-indexer following the elastic common schema and will no longer be stored in individual databases, which let us have a general overview of the vulnerabilities for all agents and not a single agent as it is today. 
In 4.7 you still can use everything was present in 4.6. 

Regards!

**Comment by sebasfalcone** [comment]
Inactive thread, fell free to re open it if necessary

**Comment by Rudiedevries** [comment]
I am trying to pull vulnerability details via API to MSSQL to build reports with historical data, but now this API call has been removed.
How can I pull these details now?

**Reply by MiguelazoDS** [reply]
Hi @Rudiedevries, 

As vulnerabilities are directly indexed into wazuh-indexer you can use the OpenSearch API to get the vulnerabilities. 
Please take a look at https://opensearch.org/docs/latest/api-reference/search/

Regards!

**Comment by Rudiedevries** [comment]
Thank you for your response on this.

Do you perhaps have any documentation on the api calls available for vulnerabilities on OpenSearch
Regards

[Logo]<https://www.sifutures.com/>
Rudie de Vries | Hosting Services Manager
SI Futures | www.sifutures.com<https://www.sifutures.com/>
***@***.******@***.***>
Cell: +27 84 869 5995
Office: +27 21 976 8133
[linkedin icon]<https://www.linkedin.com/company/si-futures>
***@***.***?anonymous&ep=signature>
Book time to meet with ***@***.***?anonymous&ep=signature>

**Reply by MiguelazoDS** [reply]
Hi @Rudiedevries, 

The only change respecting the link I shared before is the index name. You could use something like this to query the vulnerabilities 

`GET /wazuh-states-vulnerabilities-*/_search`

Also, you may want to use the "size" parameter too, because by default it only returns 10 entries. 

`GET /wazuh-states-vulnerabilities-*/_search?size=200`

**Comment by pzafar1974** [comment]
Hello.

If this change stops reporting vulnerability alerts in the alert.json file. I use Splunk Forwarder instead of Indexer. I am not able to see vulnerability alerts in Splunk.

Regards
